coalib/parsing/ConfParser.py

```diff
@@ -114,6 +114,13 @@ def __parse_lines(self, lines, origin):
                 current_keys = keys
 
             for section_override, key in current_keys:
+                if(key == 'bears') and value:
+                    bears = value.split(',')
+                    bears = [Bear.strip() for Bear in bears]
+                    bears = [Bear + 'Bear' if Bear and not Bear.endswith('Bear')
+                             else Bear for Bear in bears]
```

Comment:
You can do ``bears = [Bear.strip() + 'Bear' if not Bear.endswith('Bear') else Bear for Bear in value.split(',')]``
This way you avoid three times assignment to ``bear``.